app/experimenter/visualization/tests/api/test_views.py

```diff
@@ -47,11 +47,10 @@ def test_analysis_results_view_no_data(self, status, mock_exists):
         json_data = json.loads(response.content)
         self.assertEqual(
             {
-                "daily": [],
-                "metadata": [],
-                "weekly": {},
-                "overall": {},
-                "other_metrics": {},
+                "daily": None,
+                "metadata": None,
+                "weekly": None,
+                "overall": None,
                 "show_analysis": False,
             },
```

Comment:
Is this the only test change we need to capture the failures this addresses?

Reply:
I believe so. All the failures were happening because the frontend was expecting None values instead of empty objects.

Reply:
Sounds good 👍